flink-runtime/src/main/scala/org/apache/flink/runtime/akka/AkkaUtils.scala

```diff
@@ -216,12 +219,19 @@ object AkkaUtils {
    * identified by hostname.
    *
    * @param configuration instance containing the user provided configuration values
-   * @param hostname of the network interface to listen on
+   * @param hostname of the network interface to bind on
    * @param port to bind to or if 0 then Akka picks a free port automatically
+   * @param externalHostname The host name to expect for Akka messages
+   * @param externalPort The port to expect for Akka messages
    * @return Flink's Akka configuration for remote actor systems
    */
   private def getRemoteAkkaConfig(configuration: Configuration,
-                                  hostname: String, port: Int): Config = {
+                                  hostname: String, port: Int,
+                                  externalHostname: String, externalPort: Int): Config = {
+
+    LOG.info(s"Using binding address $hostname:$port" +
```

Comment:
Not too bad here, but in general, I think having loggers in util classes is not a very good idea. It typically leads to the log lines showing up very often and at strange places. The information should usually be logged by the caller of the functions.